File: PyCode/oval.py
err = 0.1
step = 0.1
def f():
	x = -5.0
	y = -3.0
	while y < 3.0:
		while x < 5.0:
			if(abs(x*x - x*y + y*y - 3) < err):
				print('x')
				print(x)
			else:
				pass
				#print(abs(x*x - x*y + y*y - 3))
				#print('')
			x = x + step
		#print('\n')
		x = 0.0
		y = y + step

if __name__ == '__main__':
	f()

File: ccode/oval.c
float err = 0.1f;
float step = 0.01f;
int main(int argc, char const *argv[])
{
	for (float y = -3.0f; y < 3.0f; y+=step)
	{
		/* code */
		float x;
		for (x = -3.0f; x < 3.0f; x+=step)
		{
			/* code */
			if(abs(x*x - x*y + y*y - 3.0f) < err) {
				printf("x");
				exit(0);
			}
			else printf(" ");
		}
		printf("\n%f",x*x - x*y + y*y - 3.0f);
	}
	return 0;
}